src/Properties/ObjectProperty.js

```diff
@@ -36,13 +33,17 @@ const ObjectProperty = React.forwardRef(
             </Button>
           </Box>
           {value && (
-            <Box border={{ side: 'horizontal' }}>
+            <Box flex={false} border={{ side: 'bottom' }}>
               <Button
-                icon={<FormClose />}
                 tip={`clear ${name}`}
                 hoverIndicator
                 onClick={() => onChange(undefined)}
-              />
+                pad={{ vertical: 'xsmall', horizontal: 'small' }}
+              >
+                <Box pad={{ vertical: 'xsmall', horizontal: 'small' }}>
+                  <FormClose />
+                </Box>
+              </Button>
```

Comment:
Can we change this to
```
<Button
  icon={<FormClose />}
  tip={`clear ${name}`}
  hoverIndicator
  onClick={() => onChange(undefined)}
  plain
  fill="vertical"
/>
```
Then I think we don't need the changes to `flex` on the other lines